docs/content/en/docs/concepts/architecture/components/metrics-operator/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Keptn Metrics Operator
+linktitle: Metrics Operator
+description: Basic understanding of Keptn's Metrics Operator
+weight: 80
+cascade:
+---
+
+
+**Metrics Operator** collects, processes, and analyzes metrics data from a variety of sources. Once collected, this data, can be used to generate a variety of reports and dashboards that provide insights into the health and performance of the application and infrastructure.
+
+The Metrics Operator consists of two main components:
+
+* Metrics Controller
+* Metrics Adapter
+
+```mermaid
+graph TD;
+    Metrics-Operator-->Metrics-Adapter;
+    Metrics-Operator-->Metrics-Controller
+style Metrics-Operator fill:#006bb8,stroke:#fff,stroke-width:px,color:#fff
+style Metrics-Adapter fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Metrics-Controller fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+```
+
+**Metrics adapter** is used to expose custom metrics from an application to external monitoring and alerting tools. The adapter exposes custom metrics on a specific endpoint where external monitoring and alerting tools can scrape them. It is an important component of the metrics operator as it allows for the collection and exposure of custom metrics, which can be used to gain insight into the behavior and performance of applications running on a Kubenetes cluster.     
+
+**Metrics controller** is used to fetch metrics from a SLI provider. The controller reconciles a `KeptnMetric` CR and updates it's status with the metric value provided by the selected SLI provider. The steps in which the Controller fetches metrics are given below:
```

Comment:
Make "KeptnMetric" a link to https://main.lifecycle.keptn.sh/docs/yaml-crd-ref/metric/ .  Yes, brand new page that doesn't have content but will soon.

Reply:
I would use a partial link: `../../../../yaml-crd-ref/metric`. This way, we don't depend on the website name :)